lib/tether/ops_linux.go

```diff
@@ -32,6 +31,7 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/d2g/dhcp4"
+	dockerUser "github.com/opencontainers/runc/libcontainer/user"
```

Comment:
Not the fan of idea having libcontainer deps in the tether. I know you had issues with https://golang.org/pkg/os/user/ so please document them here.

Also why we refer this package as dockerUser? We could drop that and use user.XYZ